app/scrapers/scraper.rb

```diff
@@ -26,6 +34,9 @@ def scrape
     end
   end
 
+  # @return [Array] an Array of objects most likely retrieved via an API call.
+  #   Each element will get passed to the scraper template methods in order to
```

Comment:
@rathrio: Maybe we should elaborate what is actually meant by _templete methods_ .